### DISCUSSION TITLE
How can I package support for a language?

### DISCUSSION THREAD

**Discussion by Jummit** [helix-editor/helix#10450] [Q&A]
I have a language which is used internally, and would like to distribute support for Helix in a robust and cross-platform way. It would be nice if HELIX_RUNTIME would supported multiple paths, then one could add the folder with language-specific configuration to HELIX_RUNTIME.

Example for Linux:

```sh
export HELIX_RUNTIME=${HELIX_RUNTIME:-${XDG_CONFIG_HOME:-$HOME/.config}/helix/runtime}:/language-support/runtime

# or, in short

export HELIX_RUNTIME=~/.config/helix/runtime:/language-support/runtime
```

Currently, I'd have to edit the `languages.toml` file and modify the (only) runtime folder, which will be a little difficult to maintain.

Any ideas on how to best package Helix configuration?

**Comment by the-mikedavis** [comment]
I think you might be able to use `HELIX_DEFAULT_RUNTIME` for this if you're also building Helix https://github.com/helix-editor/helix/pull/8610

**Reply by Jummit** [reply]
Hm, I'm not providing the Helix executable, that should be installed on each machine using the package manager. But this wouldn't scale well either, with support for just one "package".

**Reply by the-mikedavis** [reply]
Internally we allow multiple runtime directories, see https://github.com/helix-editor/helix/blob/ab203b5f53426ab0701d18aff1a6c6370c080241/helix-loader/src/lib.rs#L30-L77. It could be useful to allow specifying more than one with `HELIX_RUNTIME` like you say, and I think it could be a very small change
